packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -17,6 +17,12 @@ import { errorCodes } from '@metamask/rpc-errors';
 import HttpProvider from 'ethjs-provider-http';
 import NonceTracker from 'nonce-tracker';
 
+import { FakeBlockTracker } from '../../../tests/fake-block-tracker';
+import { mockNetwork } from '../../../tests/mock-network';
+import type {
+  AcceptResultCallbacks,
+  AddResult,
+} from '../../approval-controller/src';
```

Comment:
Assuming this is exported:
```suggestion
} from '@metamask/approval-controller';
```